frontend/src/features/moti/mapSideBar/forms/PropertySearchForm.tsx

```diff
@@ -0,0 +1,172 @@
+import { ReactComponent as ParcelDraftIcon } from 'assets/images/draft-parcel-icon.svg';
+import { Input } from 'components/common/form';
+import SearchButton from 'components/common/form/SearchButton';
+import { Label } from 'components/common/Label';
+import { pidFormatter } from 'features/properties/components/forms/subforms/PidPinForm';
+import { GeocoderAutoComplete } from 'features/properties/components/GeocoderAutoComplete';
+import { Formik, getIn } from 'formik';
+import { IGeocoderResponse } from 'hooks/useApi';
+import { useState } from 'react';
+import { Col, Form, Row } from 'react-bootstrap';
+import ClickAwayListener from 'react-click-away-listener';
+import styled from 'styled-components';
+
+const SearchMarkerButton = styled.button`
+  top: 20px;
+  right: 20px;
+  border: 0px;
+  background-color: white;
+`;
+
+const SearchForm = styled(Row)`
+  .form-row {
+    margin: 0.5rem;
+    justify-content: flex-end;
+  }
+  .label {
+    margin: 0;
+  }
+  .form-group {
+    margin: 0 0.5rem;
+  }
+  .btn {
+    min-height: 20px;
+  }
+  .GeocoderAutoComplete {
+    width: auto;
+  }
+`;
+
+const SearchHeader = styled.h5`
+  text-align: left;
+`;
+
+interface ISearchFormProps {
+  /** used for determining nameSpace of field */
+  nameSpace?: string;
+  /** handle the population of Geocoder information */
+  handleGeocoderChanges: (data: IGeocoderResponse, nameSpace?: string) => Promise<void>;
```

Comment:
Why is `handlePidChange` singular, but `handleGeocoderChanges` plural?

Reply:
fixed in follow-up pr.